wiki/Tournaments/RMoT_Invitational/5/en.md

```diff
@@ -0,0 +1,166 @@
+| Referees | ![][flag_PH] [Deadpulse](/users/8575527 "Deadpulse"), ![][flag_DE] [DungeonFace](/users/1991031 "DungeonFace"), ![][flag_US] [Elcheer](/users/4420014 "Elcheer"), ![][flag_PL] [HAEN24](/users/4390077 "HAEN24"), ![][flag_SG] [Kumabread](/users/7210491 "Kumabread"), ![][flag_PL] [MagBlood](/users/6178640 "MagBlood"), ![][flag_RU] [NikerVM](/users/3618495 "NikerVM"), ![][flag_RU] [Soquire](/users/3618495 "Soquire"), ![][flag_GB] [Stedoss](/users/8331546 "Stedoss"), ![][flag_US] [YoshiLover456](/users/6843383 "YoshiLover456") |
+
+## Prizes
+
+The prizes are handed out in the form of **points**; every 48 points can be redeemed for a month of osu!supporter.
+
+| Placing       | Prize                     |
+| ------------: | :------------------------ |
+| First Place   | 96 Points & profile badge |
+| Second Place  | 48 Points                 |
+| Third Place   | 24 Points                 |
+| Fourth Place  | 12 Points                 |
+
+### Predictions
+
+Anyone was allowed to predict the outcome of the tournament until the starting date. The predictions took place on Challonge and the top three predicters were awarded one fourth the amount awarded to the players:
+
+| Placing      | Prize     |
+| -----------: | :-------- |
+| First Place  | 24 Points |
+| Second Place | 12 Points |
+| Third Place  | 6 Points  |
+| Fourth Place | 3 Points  |
+
+![][flag_GB] [Olii](/users/7683305 "Olii") won the predictions, with ![][flag_SI] [Redavor](/users/3328606 "Redavor") in second place, and ![][flag_PL] [Malson](/users/6047395 "Malson") in third place.
+
+## Players
+
+16 players are admitted into each Invitational. Players eligible for Invitational 2 are:
+
+- ![][flag_RU] [\_Suzuha](/users/8445602 "\_Suzuha")
+- ![][flag_NO] [Bissas](/users/7618918 "Bissas")
+- ![][flag_US] [diamondturtle01](/users/4118117 "diamondturtle01")
+- ![][flag_LT] [Eismis](/users/1436793 "Eismis")
+- ![][flag_MA] [iTzAshraf](/users/10285023 "iTzAshraf")
+- ![][flag_US] [jppz](/users/7889571 "jppz")
+- ![][flag_AU] [Klein-](/users/2153933 "Klein-")
+- ![][flag_RU] [KoloMl](/users/6544403 "KoloMl")
+- ![][flag_RU] [Nogad](/users/6488077 "Nogad")
+- ![][flag_GB] [Olii](/users/7683305 "Olii")
+- ![][flag_BR] [PlayPen](/users/5094351 "PlayPen")
+- ![][flag_FR] [Prosdokies](/users/3852436 "Prosdokies")
+- ![][flag_RU] [Well Halen](/users/8660244 "Well Halen")
+- ![][flag_UA] [Xelite](/users/7793117 "Xelite")
+
+**Notice:** Klein- and Nogad gave up their places in the Invitational, and iTzAshraf and \_Suzuha reached semi-finals twice. Due to this, 4 people who have not reached semi-finals were selected to participate in the tournament: filler players.
```

Comment:
Italicize this notice completely (everything said) and remove the bold on `Notice:`.

`*Notice: Klein- and Nogad gave up their places in the Invitational, and iTzAshraf and \_Suzuha reached semi-finals twice. Due to this, 4 people who have not reached semi-finals were selected to participate in the tournament: filler players.*`